prow/cmd/deck/main.go

```diff
@@ -511,8 +511,8 @@ type podLogClient struct {
 	client corev1.PodInterface
 }
 
-func (c *podLogClient) GetLogs(name string) ([]byte, error) {
-	reader, err := c.client.GetLogs(name, &coreapi.PodLogOptions{Container: kube.TestContainerName}).Stream()
+func (c *podLogClient) GetLogs(name string, container string) ([]byte, error) {
```

Comment:
You can/should combine the types when they're the same. Something like this:
`... GetLogs(name, container string) ...`